autoload/go/doc.vim

```diff
@@ -148,6 +148,11 @@ function! s:gogetdoc(json) abort
   let pos = shellescape(fname.':#'.offset)
 
   let cmd += ["-pos", pos]
+
+  if exists('g:go_build_tags')
+    let cmd += ["-tags", get(g:, 'go_build_tags')]
```

Comment:
Did you test that this will work with multiple build tags? I'm, not 100% sure, but I think you need to do:

    ['-tags', go#util#Shellescape(get(g:, 'go_build_tags'))]

So that `g:go_build_tags='foo bar'` will be:

    gogetdoc -tags 'foo bar' someident

Instead of:

    gogetdoc -tags foo bar someIdent

Reply:
Yes, you are right, sorry for that changes. I had read the doc about build constraints again.
And I saw your commit #1705. That's much better. Thanks for your advice.